app/controllers/application_controller.rb

```diff
@@ -27,7 +27,7 @@ def configure_permitted_parameters
 	end
 
 	def set_global_info
-		@edition = Edition.find(2016)
+		@edition    = Edition.find(2017)
```

Comment:
@frmendes I love indenting it like this so it aligns with the equals sign right below 😬
But I get it, will change it :+1: